VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/ConferenceManagementControllerTestBase.cs

```diff
@@ -36,7 +39,7 @@ protected ConferenceManagementController SetupControllerWithClaims(ClaimsPrincip
             };
 
             return new ConferenceManagementController(VideoApiClientMock.Object, MockLogger.Object,
-                ConferenceCacheMock.Object)
+                ConferenceCacheMock.Object, HearingLayoutServiceMock.Object)
```

Comment:
AutoMock would be nice here but I won't put that on you to refactor as a part of this PR